Stripe/PublicHeaders/STPAPIClient.h

```diff
@@ -30,10 +30,12 @@ static NSString *const STPSDKVersion = @"18.3.0";
 @interface Stripe : NSObject FAUXPAS_IGNORED_ON_LINE(UnprefixedClass);
 
 /**
- Set your Stripe API key with this method. New instances of STPAPIClient will be initialized with this value. You should call this method as early as
+ Set your Stripe API key with this method. You should call this method as early as
  possible in your application's lifecycle, preferably in your AppDelegate.
-
- @param   publishableKey Your publishable key, obtained from https://stripe.com/account/apikeys
+ 
+ This is the same as setting STPAPIClient.shared.publishableKey.
+ 
+ @param   publishableKey Your publishable key, obtained from https://dashboard.stripe.com/apikeys
  @warning Make sure not to ship your test API keys to the App Store! This will log a warning if you use your test key in a release build.
  */
 + (void)setDefaultPublishableKey:(NSString *)publishableKey;
```

Comment:
since we're documenting to use `STPAPIClient.shared.publishableKey` instead should this just be deprecated?

Reply:
I leaned towards keeping it b/c:
- less breaking for users (esp with the above comment's change)
- no need to update docs recommendation
- `Stripe.setDefaultPublishableKey` reads better than `STPAPIClient.shared.publishableKey`